Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -757,7 +757,7 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         node,
         .standaloneSemicolonStatement,
         fixIts: [
-          FixIt(message: RemoveNodesFixIt(semicolon), changes: .makeMissing(semicolon))
+          FixIt(message: RemoveNodesFixIt(semicolon), changes: .makeMissing(semicolon, transferTrivia: false))
```

Comment:
Why do we need `transferTrivia: false` here? Shouldn’t we keep the trivia? For example the comment in the following case?

```swift
let x = 1 ; // some comment
```

Reply:
I added `transferTrivia: false` to kinda sweep a nasty issue under the rug but I forgot to fix it afterwards. I'll need to some more time to properly handle,
```swift
for 
  ;
} 
// after Fix-its
for 
  <#pattern#> in <#expression#> {
}
// desired
for <#pattern#> in <#expression#> {
}
```

Reply:
I think the `after Fix-its` result is acceptable in this case. It’s a silly example.